lib/hexpm/web/templates/dashboard/key/index.html.eex

```diff
@@ -32,6 +32,15 @@
                 <% end %>
               </td>
             </tr>
+            <tr>
+              <td colspan=3>
+                <small>
+                  Last used: <%= formatted_used_at(key.last_used_at) %><br>
+                  Last IP: <%= key.last_ip || "N/A" %><br>
+                  Last user agent: <%= key.last_user_agent || "N/A" %>
+                </small>
+              </td>
+            </tr>
```

Comment:
Can you show how it would look like if we added a "Last use" column and printed it as: "2018-06-01 09:40:00, 127.0.0.1, Hex/0.17.2"?

Reply:
Hi, it just about fits in desktop but breaks mobile I am afraid:
![image](https://user-images.githubusercontent.com/4353861/40936202-820d2e36-6832-11e8-8a06-5a8e2643a7cb.png)
![image](https://user-images.githubusercontent.com/4353861/40936373-0cc3d750-6833-11e8-8c9d-9dcb5c99262f.png)

Reply:
I think it looks okay on mobile, it is fine to line break. Please change the formatting to "2018-06-01, 127.0.0.1, Hex/0.17.2" and we are good to go.

Reply:
Hi, I have made the change although I still think the layout does not look good on small mobiles. The menu shrinks to fit the screen but the "Keys" sections becomes too wide to fit.

iPhone 6/7/8 Plus in Chrome dev tools look fine:

![image](https://user-images.githubusercontent.com/4353861/41501394-866886b6-719b-11e8-9cde-1d2af5633860.png)

However, on iPhone SE (dev tools) the menu fits on the screen but the Keys sections doesn't and can't be scrolled sideways:

![image](https://user-images.githubusercontent.com/4353861/41501403-af78a6ee-719b-11e8-905b-70b9d216df01.png)

The latter is also the case on a Huawei P10 Lite which used to test locally.

Let me know if you think it is good as it is or if you'd like to try a different approach.